assembl/templates/notifications/html_mail_post_synthesis.jinja2

```diff
@@ -9,12 +9,14 @@ notification
 frontendUrls
 ink_css
 synthesis
+jinja_env
+lang_prefs
 #}
 
 {% extends "notifications/html_mail_post.jinja2" %}
 
 {%- block post_body %}
-  {{ synthesis.as_html(jinja_env) }}
+  {{ synthesis.as_html(jinja_env, lang_prefs) }}
   {#
   {{ synthesis.introduction }}
```

Comment:
You should fix this one `synthesis.introduction` and `synthesis.conclusion` to avoid a warning/error in celery, even if it's just a comment, I guess replace it with `synthesis.introduction.best_lang(lang_prefs).value|safe` as well?

Reply:
I did not see it produced any warning or error, and does it really as it is commented code? And this was out of scope of this fix but OK I remove this legacy commented code.